### PR TITLE
Pass innerInputClassNames to Input inside Combobox

### PR DESCRIPTION
**Pull request by kkirasich**

### CODE REVIEW

**Comment by kkirasich** [comment]
[Don't automatically expand the gl account selector when visiting the first step of the bank switch page](https://trello.com/c/qNZwmjwA/1276-dont-automatically-expand-the-gl-account-selector-when-visiting-the-first-step-of-the-bank-switch-page)

File: src/components/Combobox.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const defaultProps = {
 };
 
 const Combobox: React.FunctionComponent<ComboboxProps> = ({
-  className, direction, disabled, dropdownProps, options, placeholder, value, menuMaxHeight,
+  className, direction, disabled, dropdownProps, innerInputClassNames, options, placeholder, value, menuMaxHeight,
```
Comment (gthomas-appfolio) [inline src/components/Combobox.tsx:43]: Sorry to nitpick but `inputClassName`sounds a bit cleaner

**Review by gthomas-appfolio — COMMENTED** [review]
One comment

File: src/components/Combobox.tsx (hunk 1 above)
Comment (kkirasich) [reply src/components/Combobox.tsx:43]: I agree, done!

**Review by gthomas-appfolio — APPROVED** [review]

**Pull request merged**